### PR TITLE
redesigned database

### PR DESCRIPTION
**Pull request by 2naCan**

### CODE REVIEW

**Comment by JBHarrison5** [comment]
nice one, delete index.php/work on it

File: gameCollection_2024-02-26.sql [hunk 1]
```diff
@@ -29,27 +29,103 @@ CREATE TABLE `gameDisplay` (
...
-DROP TABLE IF EXISTS `playerLibrary`;
+DROP TABLE IF EXISTS `userData`;
 
-CREATE TABLE `playerLibrary` (
+CREATE TABLE `userData` (
   `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
-  `steamID` int(20) NOT NULL,
-  `gameCount` int(11) DEFAULT NULL,
+  `rating` int(11) DEFAULT NULL,
+  `review` int(11) DEFAULT NULL,
```
Comment (matt-ioa) [inline gameCollection_2024-02-26.sql]: Why is `review` an integer?

Comment (2naCan) [reply gameCollection_2024-02-26.sql]: thank you :)

File: userLibrary.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<?php
```
Comment (matt-ioa) [inline userLibrary.php:1]: Maybe add a couple of comments to explain what this file does

**Review by matt-ioa — CHANGES_REQUESTED** [review]
Couple of comments, see what you think

**Pull request closed without merging**